pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -153,8 +153,10 @@ type RolloutExperimentStep struct {
 	// Duration is the duration in seconds that the experiment should run for
```

Comment:
Shouldn't Templates of type []RolloutExperimentTemplate in RolloutExperimentStep the two lines above this comment have a patchStrategy?